app/controllers/claims/pages_controller.rb

```diff
@@ -6,4 +6,39 @@ class Claims::PagesController < Claims::ApplicationController
   before_action :redirect_to_after_sign_in_path, only: :start, if: :user_signed_in?
 
   def start; end
+
+  def grant_conditions
+    @breadcrumbs = {
+      t("home") => claims_root_path,
+      t(".grant_conditions.page_title") => claims_grant_conditions_path,
+    }
+  end
```

Comment:
Personally, I'm not a fan of UI setup in controller code, I'd rather set it up in the views directly.

Not a blocker though.

```erb
<%= govuk_breadcrumbs(breadcrumbs: { t("home") => claims_root_path, t(".grant_conditions.page_title") => claims_grant_conditions_path }) %>
```

Reply:
Yeah, not sure. I would put stuff like this in a decorator but there's nothing to decorate and the views are loaded with content. If we go with a partial we can't put the paths in the view